lib/foreman_tasks/dynflow/daemon.rb

```diff
@@ -68,5 +87,43 @@ def run_background(command = 'start', options = {})
     def world
       ForemanTasks.dynflow.world
     end
+
+    private
+
+    def daemons_options(command, options)
+      {
+	:multiple => true,
+	:dir => options[:pid_dir],
+	:log_dir => options[:log_dir],
+	:dir_mode => :normal,
+	:monitor => true,
+	:log_output => true,
+	:log_output_syslog => true,
+        :monitor_interval => options[:memory_polling_interval] / 2,
```

Comment:
The indetantion seems odd. Except from that, what's the reason for introducing this setting and making that dependent on memory_polling_interval?